src/Resources/config/listener.yml

```diff
@@ -0,0 +1,89 @@
+##
+# Listener priorities
+#
+# The priorities of the following listeners must be lower than the one of the
+# Symfony route listener (defaults to 32).
+#
+# - 30: ContainerScopeListener
+# - 28: RefererIdListener
+# - 26: InitializeSystemListener
+# - 24: ToggleViewListener
+# - 22: OutputFromCacheListener
+#
+# Session listener
+#
+# The priority of the session listener must be higher than the other listeners but
+# lower than the one of the Symfony session listener (defaults to 128).
+#
+# - 64: SessionListener
+#
+# Do not change the priorities unless you know what you are doing!
+##
+services:
+    contao.listener.add_to_search_index:
+        class: Contao\CoreBundle\EventListener\AddToSearchIndexListener
+        tags:
+            - { name: kernel.event_listener, event: kernel.terminate, method: onKernelTerminate }
+
+    contao.listener.container_scope:
+        class: Contao\CoreBundle\EventListener\ContainerScopeListener
+        arguments:
+            - "@service_container"
+        tags:
+            - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest, priority: 30 }
+            - { name: kernel.event_listener, event: kernel.finish_request, method: onKernelFinishRequest, priority: -254 }
```

Comment:
It will not be guaranteed to be the last one, no matter what priority we choose (except [`-INF`](http://php.net/manual/en/math.constants.php#constant.inf) ^^. The [Symfony documentation](http://symfony.com/doc/current/cookbook/service_container/event_listener.html) says to choose a priority between +255 and -255

> There is an additional tag option priority that is optional and defaults to 0. This value can be from -255 to 255, and the listeners will be executed in the order of their priority (highest to lowest). This is useful when you need to guarantee that one listener is executed before another.

Though that's not really correct, as it can be any integer value.